src/libopensc/card-starcos.c

```diff
@@ -268,7 +406,7 @@ static int process_fci(sc_context_t *ctx, sc_file_t *file,
 	return SC_SUCCESS;
 }
 
-static int process_fci_v3_4(sc_context_t *ctx, sc_file_t *file,
+static int process_fci_v3_2_v3_4_v3_5(sc_context_t *ctx, sc_file_t *file,
```

Comment:
maybe renaming this to `process_fci_v3` would be useful

Reply:
Starcos 3.1 and 3.6 are not included, that's the reason for this naming scheme. Maybe process_fci_v3_2-5 ? Or solely put this information in a comment above the function and name it process_fci_v3?

Reply:
there is no 3.1 or 3.6 at the moment, hence there is only 3.x and everything else. just change the naming.